### ISSUE TITLE
Urban Jungle (Phase 2, Map 16) violates the ears upon finishing

### ISSUE THREAD

**Issue by Bracket-H** [freedoom/freedoom#1161]
Please, this wasn't too good for my headphones either, don't potentially wreck hardware please... 

That city explosion ...perhaps tone it down a little, or if its a prefab sound (then what's wrong with you recording/making such a thing?!) ...then change it out please.

Ow...

**Comment by unithecutie** [comment]
This was addressed in the Freedoom Discord server! It's using the boss brain death sound.

**Comment by Calinou** [comment]
I agree the boss brain pain/death sounds are really loud in general – they seem quite a bit louder and crunchier than their Doom 2 counterparts.

**Comment by unithecutie** [comment]
https://github.com/freedoom/freedoom/assets/139153158/e7035932-c6e3-41f5-85f2-10234f7d3992

**Comment by mc776** [comment]
I've taken the liberty of just making this issue about the boss brain death sound generally.

**Comment by mc776** [comment]
Reverting that rename now that I've been doing more (normal) boss brain stuff.

The big issue here is that we've got the 3 boss brains under slow crushers so we're getting all 3 pain sounds stuttering once per tic before the actual death sound. Any source port that properly mixes all three is going to make this much worse.

I've made a variant that only has a single boss brain a bit further up, which itself is free from the crushes and only takes damage from the barrels. There's also a hidden barrel that creates an explosion sound to foreshadow the main event a bit.
[map16.wad.zip](https://github.com/freedoom/freedoom/files/13417301/map16.wad.zip)
